dist/templates/ovnkube-identity.yaml.j2

```diff
@@ -49,6 +49,17 @@ spec:
                     operator: In
                     values:
                       - "linux"
+        podAntiAffinity:
```

Comment:
was that part always supposed to be here, but we never noticed because we only use single-master deployments?

Reply:
Yes, that is very plausible. When this is deployed via CNO, we have this section when replicas is greater than 1, but it does not hurt to always have it:
https://github.com/openshift/cluster-network-operator/blob/1daca87a5c6ebd5e6d7b5bcc4374119b59061af6/bindata/network/node-identity/managed/node-identity.yaml#L66-L81